config/sync/layout_library.layout.news_spotlight.yml

```diff
@@ -147,34 +147,21 @@ layout:
               field_label:
                 label_value: ''
                 label_tag: ''
-        weight: 9
+        weight: 10
         additional: {  }
-      b9778f05-c8e7-4f90-8456-3d0125d36538:
-        uuid: b9778f05-c8e7-4f90-8456-3d0125d36538
+      1067a8ed-6ab9-401a-a6df-fe6c069d4a08:
+        uuid: 1067a8ed-6ab9-401a-a6df-fe6c069d4a08
         region: left
         configuration:
-          id: 'field_block:node:stanford_news:layout_selection'
```

Comment:
The layout selection wasn’t appearing for users who are not logged in. I tested a few things, and I’m guessing it’s because the layout selection is limited to logged-in users.

@pookmish would this be a reasonable workaround to adding the "Spotlight" text?